api/tests/Api/SnapshotTests/__snapshots__/ResponseSnapshotTest__testOpenApiSpecMatchesSnapshot__1.yml

```diff
@@ -632,15 +674,6 @@ components:
       deprecated: false
       description: 'A piece of programme that will be carried out once or multiple times in a camp.'
       properties:
-        _links:
-          properties:
-            self:
-              properties:
-                href:
-                  format: iri-reference
-                  type: string
-              type: object
```

Comment:
Any idea why this vanished?

Reply:
Sadly no, but it doesn't affect us.
I can raise an issue with api platform/openapi later